CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 0.28.0
```

Comment:
I think you should leave this as `Unreleased`, I had a problem once with a release when I edited this manually.